### ISSUE TITLE
Portmaster drops all connections when the OS is started without connectivity.

### ISSUE THREAD

**Issue by Marc05** [safing/portmaster#1064]
In Windows 11 22H2, if is started when the gateway is down (and hence no DHCP), Portmaster drops all connections. Editing the adapter and entering in manual settings did not allow it to connect with other devices on the same subnet. Shutting down Portmaster allowed connections to work, and the OS connected to the internet once the gateway connectivity was restored. Re-opening Portmaster resulted in the issue returning until the OS was restarted.

**Comment by Raphty** [comment]
I tried this with my windows 11 machine, and I could not reproduce the issue.

@dhaavi can maybe chime in here, but I don't think that Portmaster needs an IP address to work, it might expect an Internet connection, but this is weird since on any laptop with no wifi connection Portmaster works and after connecting to an wifi it keeps working.

If I understand you correctly:
- your gateway is down
- so no Internet connection
- device starts
- you can't connect to other devices on the network even with a configured static ip
- you shot down Portmaster (core service and all)
- now connections work
- you start Portmaster again
- connections don't work anymore
- you restart the pc
- all works well

I think this might have something to do with Portmaster expecting an Internet connection, but I would need the debug info to look deeper into this.

https://docs.safing.io/portmaster/faq#how-do-i-share-debug-information-and-logs

**Comment by Marc05** [comment]
I suppose this can be closed given that I'm not able to reproduce it anymore. I have a feeling it had something to do with the `Other Connections` profile. Unless that profile is set to `Allow` connections, it will block things from the shell such as ping.

I ran into some other odd behavior while trying to reproduce this - changing `Default Network Action`to block/allow for `Other Connections` was not having any effect until some combination of reloading the page by changing between profiles/apps, toggling the `Block Connections` toggle, and resetting the option by clicking the `App specific configuration` icon. The last odd behavior I've noticed is that the `Copy Debug Info` option is not consistent - after copying it once, subsequent copies fail (nothing is copied to clipboard). Sometimes it does work fine (I'm on 1.0.5).

Apologies, I know none of this is very helpful in narrowing the issue down.
